specs/latest/2.0/index.html

```diff
@@ -1073,20 +1170,32 @@ <h2 class="no-toc">Table of contents</h2>
         See <a href="#COPYING_BUFFERS">Copying Buffers</a> for restrictions imposed by the WebGL 2 API.
       </dd>
       <dt>
-        <p class="idl-code">void getBufferSubData(GLenum target, GLintptr offset, ArrayBuffer? returnedData)
+        <p class="idl-code">
+          void getBufferSubData(GLenum target, GLintptr srcByteOffset,
+                                BufferDataSource? dstBuffer,
+                                optional unsigned long dstOffset = 0,
+                                optional unsigned long length = 0)
```

Comment:
There's no reason to support this being nullable, just as there's no support for glReadPixels(nullptr).